### PR TITLE
Поправил структуры, работают

### PR DESCRIPTION
**Pull request by sartfromua**
Какие параметры классов нужно изменить/добавить?

### CODE REVIEW

**Review by mecchmatProjects — CHANGES_REQUESTED** [review]
look at the comments on previous pull request

File: main.c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "structures.h"
+
+int main() {
+    Employee emp;
+    emp.person.birthday.day = 10;
```
Comment (mecchmatProjects) [inline main.c]: try to use functions

File: structures.c [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
```
Comment (mecchmatProjects) [inline structures.c]: replace this to .h file

File: structures.h [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "structures.c"
```
Comment (mecchmatProjects) [inline structures.h]: delete #include "structures.c"

here is the place for your functions implementatiions

File: structures.c (hunk 2 above)
Comment (mecchmatProjects) [reply structures.c]: #include ".....h"
Also it is better to split each structure in separate file, U can do it later...

File: structures.h (hunk 3 above)
Comment (sartfromua) [reply structures.h]: Tried deleting, main.c isn't working without #include "structures.c" in this header.

Comment (mecchmatProjects) [reply structures.h]: becouse U compile it wrong

Comment (mecchmatProjects) [reply structures.h]: How are U create project? Do U have Makefile?

Comment (mecchmatProjects) [reply structures.h]: read LectC.pdf - last chapter

Comment (sartfromua) [reply structures.h]: Codeblocks created project for me, having "project file" in folder

File: structures.h [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+
+typedef struct {
+    Employee employee;
+} Head;
+
+
+
+
+date inputDate();
```
Comment (mecchmatProjects) [inline structures.h:55]: It is much better than it was.
However U'd better split not by functionality, but by classes: Empoyee.h. Empoyee.c, Product.h, Product.c etc

**Pull request merged**